tests/test_api.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+import pyux
+import keras
+import json
+import os
+
+import keras.backend.tensorflow_backend
+import keras.backend.theano_backend
+import keras.backend.cntk_backend
+import keras.backend.numpy_backend
+import keras.utils.test_utils
+
+
+def test_api():
+    api_file = os.path.join(os.getcwd(), 'api.json')
+
+    with open(api_file, 'r') as f:
+        previous_api = json.load(f)
+
+    current_api = pyux.sign(keras)
```

Comment:
Style nit: there's a lot of whitespace in this function, you can group semantically related lines into paragraph.

Reply:
done